colossalai/auto_parallel/tensor_shard/node_handler/getatrr_hadnler.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, List
+
+from ..sharding_strategy import OperationData, OperationDataType
+from .node_handler import NodeHandler
+from .strategy import GetattrGenerator, StrategyGenerator
+
+__all__ = ['GetattrHandler']
+
+
+class GetattrHandler(NodeHandler):
+    """
+    A GetattrHandler which deals with the sharding strategies for Getattr Node.
+    """
+
+    def get_strategy_generator(self) -> List[StrategyGenerator]:
+        op_data_mapping = self.get_operation_data_mapping()
+        generators = []
+        generators.append(GetattrGenerator(op_data_mapping, self.device_mesh))
+        return generators
+
+    def get_operation_data_mapping(self) -> Dict[str, OperationData]:
+        # use transposed shape for strategies
+        # the strategies will be transformed back to its original shape in self.post_process
+        physical_output = OperationData(name=str(self.node), type=OperationDataType.OUTPUT, data=self.node._meta_data)
```

Comment:
Is it possible that the output is not a tensor?

Reply:
I see, It could be any type if users define some attributes with random types to module.

Reply:
There are only two possible types for get_attr node:
1. torch.Tensor( torch.nn.Parameters)
2. torch.nn.Module
Temporarily, we just support first case in our tracer, so we don't have to worry about issue caused by output type.